application-load-balancer/main.tf

```diff
@@ -314,7 +307,7 @@ data "aws_iam_policy_document" "origin_verify_rotate_policy" {
     effect  = "Allow"
     actions = ["wafv2:*"]
     resources = [
-      aws_wafv2_web_acl.waf-acl.arn
+      aws_wafv2_web_acl.waf-acl[""].arn
```

Comment:
Do we need the `[""]` if there is only ever going to be none or one resource? It looks odd. Question applies elsewhere too.

Reply:
I think you're right, but I wish you were not. It's sad, because it creates the need for all those moved blocks when we only want one of these resources anyway.

Reply:
It can be done using a count instead, but you'd just end up with `[0]` instead. It's a workaround for no conditionals in terraform.